spec/spec.md

```diff
@@ -139,6 +139,19 @@ These features are deprecated, but can be supported by wallet endpoints to maint
 
 - [`solana:signTransactions`](#sign_transactions)
 
+## Account Profiles
+
+Account profiles are labels that are used to decribe an authorized account's capabilities and intended use. Each authorized account that is returned in [`authorize`](#authorize) request response can include a list of these profiles. The dapp endpoint can use these labels in combination with the account's list of [`features`](#feature-identifiers) to determine how each acount should be used. 
+
+- `primary`
```

Comment:
Does `primary` imply mutability, or would you always expect this to be accompanied by a mutability label?

Reply:
My thought is, primary will not imply mutability.